components/widgets/bitbucket/pull-request-count.js

```diff
@@ -40,10 +42,15 @@ export default class BitbucketPullRequestCount extends Component {
   }
 
   async fetchInformation () {
-    const { url, project, repository, users } = this.props
+    const { authKey, url, project, repository, users } = this.props
+    let opts = {}
```

Comment:
wouldn´t be 
```
const ops = authKey ? { headers: basicAuthHeader(authKey) } : {}
````
a little bit nicer?
or create a helper function that return the opts (we have this check in every widget)